tests/test_blobstore.py

```diff
@@ -142,21 +142,19 @@ def _put_local_obj(self, bucket: str, data: bytes, key: Optional[str]=None) -> s
             fh.write(data)
         return path
 
-    def test_s3_multipart_writer(self):
+    def test_multipart_writers(self):
         expected_data = test_data.multipart
-        with self.subTest():
-            chunk_size = get_s3_multipart_chunk_size(len(expected_data))
-            number_of_chunks = ceil(len(expected_data) / chunk_size)
-            key = f"{uuid4()}"
-            start_time = time.time()
-            with S3MultipartWriter(s3_test_bucket, key) as writer:
-                for i in range(number_of_chunks):
-                    part = Part(number=i,
-                                data=expected_data[i * chunk_size: (i + 1) * chunk_size])
-                    writer.put_part(part)
-            print("upload duration", time.time() - start_time)
-            retrieved_data = aws.resource("s3").Bucket(s3_test_bucket).Object(key).get()['Body'].read()
-            self.assertEqual(expected_data, retrieved_data)
+        chunk_size = get_s3_multipart_chunk_size(len(expected_data))
```

Comment:
Should `get_s3_multipart_chunk_size` be renamed to `get_multipart_chunk_size`?

Reply:
That chunk size calculation is intimately related to S3. Since it's possible to use a different chunk size for GS destinations, I'd prefer to use the current name.